### ISSUE TITLE
Left/right overbank length is not calculated

### ISSUE THREAD

**Issue by schoeller** [erpas/rgis#38]
Dear all,

thanks for this nice product at first.

I am having an issue with overland banks not being calculated in some parts along a reach. Flowpaths have been defined in direction of flow with station 0 being downstream. In direction of flow banks have been defined left and right. Cross-sections overlap the flowpath boundaries. 

Geometries may look like this
![image](https://user-images.githubusercontent.com/106036/45817880-98a4f600-bccf-11e8-898f-c4a3ec12de99.png)

In this case station 2376 is not given a LLength nor RLength, but ChLength, LeftBank and RighBank are determined. The station downstream at 2266 has all values calculated. 

I am using Qgis 3.2.3 with. When calling "About" no Window is opening. Extension manager is saying version 3.0 for RiverGIS plugin.

Any hints greatly appreciated.

Best

Sebastian

**Comment by luchmhor** [comment]
As far as I found out lines of type "Right" and "Left" are missing. You can go and edit "Flowpaths" by hand. You'll see type "channel" in there. This is used to calculate the center length of the Cutlines. So you can just go ahead an insert new features of type "Left" and "Right" at the end of the XSCutlines. It worked for me in most cases, some values are still 0 (don't know why).

**Comment by ichsan2895** [comment]
Yes, I just face that issue, Somehow, LOB and ROB have calculated but zero value in main stream channel.
The solution is making third flowpath line, so it has Right, Channel, Left for line types. You can copy third flowlines from "StreamCenterLines".

![RiverGIS_5](https://user-images.githubusercontent.com/60387342/100172495-0a55db80-2efb-11eb-87ec-31aae1d07f20.PNG)
![RiverGIS_6](https://user-images.githubusercontent.com/60387342/100172500-0cb83580-2efb-11eb-9279-4e658873932c.PNG)
